adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -250,7 +267,7 @@ - (BOOL)hasDelegateForPlacementID:(nonnull NSString *)placementID
 - (BOOL)canRequestBannerAdForPlacementID:(nonnull GADMAdapterVungleBannerRequest *)bannerRequest {
   @synchronized(_bannerDelegates) {
     if (_bannerDelegates.count > 0) {
-      return _bannerPlacementID == nil || [_bannerDelegates objectForKey:bannerRequest];
+      return _bannerRequest == nil || [_bannerRequest isEqualToBannerRequest:bannerRequest];
```

Comment:
_bannerRequest should never be nil, right? Because what's passed in is nonnull. A nil _bannerRequest would signify a bigger issue and shouldn't request ads.

Reply:
@ericleich  Hi Eric, when a Banner finished displaying ad such as users close the Banner ad view or exit the VC displays a Banner ad, _bannerRequest should be nil. Then adapter can request and display another Banner ad that has a different bannerRequest. Thanks a lot. :)